Modules/localStorage.js

```diff
@@ -0,0 +1,15 @@
+export function loadLocalStorage() {
+  if (localStorage.getItem('book') === null) {
+    localStorage.setItem('book', JSON.stringify([]));
+    const data = localStorage.getItem('book');
+    const bookStorage = JSON.parse(data);
+    return bookStorage;
+  }
+  const data = localStorage.getItem('book');
+  const bookStorage = JSON.parse(data);
+  return bookStorage;
+}
+
+export function updateLocalStorage(data) {
+  localStorage.setItem('book', JSON.stringify(data));
+}
```

Comment:
- [ ] as mentioned in the requirements you should Use ES6 syntax, this includes making use of arrow functions instead of the `function` keyword.